string view_mode
---
string status
